### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Ar4ik12**
x Header height is set in 1 place (for the links)
x Content is vertically centered (for any header height)
x CSS is used to show all letters in Uppercase (don't type them in HTML)
x Logo is an image wrapped with a link
x CSS Variable is used for a blue color
x Pseudo-element is used for a blue line below the active link
x Code follows all the [Code Style Rules ❗️](https://github.com/Ar4ik12/layout_moyo-header/blob/master/checklist.md)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,73 @@
...
+.is-active::after {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  content: "";
+  height: 4px;
+  background: #00acdc;
+  border-radius: 8px;
+  /* opacity: 0; */
```
Comment (OleziO) [inline src/style.css:72]:
```suggestion
```

**Review by OleziO — CHANGES_REQUESTED** [review]
Add demo link to PR description

**Comment by Ar4ik12** [comment]
> x Header height is set in 1 place (for the links) x Content is vertically centered (for any header height) x CSS is used to show all letters in Uppercase (don't type them in HTML) x Logo is an image wrapped with a link x CSS Variable is used for a blue color x Pseudo-element is used for a blue line below the active link x Code follows all the [Code Style Rules ❗️](https://github.com/Ar4ik12/layout_moyo-header/blob/master/checklist.md)

- [DEMO LINK](https://Ar4ik12.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Ar4ik12.github.io/layout_moyo-header/report/html_report/)

**Comment by Ar4ik12** [comment]
[DEMO LINK](https://Ar4ik12.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://Ar4ik12.github.io/layout_moyo-header/report/html_report/)

**Pull request closed without merging**